packages/styles/components/_c.button.scss

```diff
@@ -25,7 +26,6 @@
   &--full {
     @include modify-from-screens($major-screens) {
```

Comment:
As the code is today it will not work properly for a BU who is not using the mqpacker on demand.

The problem without mqpacker is that responsive modifier declarations are not ordered in a good way. But you didn't change declarations order. Is it normal ? How, as a BU that is not using mqpacker, can i be sure that responsive modifier will work properly if we don't change the declaration order ?